From e30315182c5d95e6f5c14b4b743504434f966edd Mon Sep 17 00:00:00 2001
From: Radostin Stoyanov <rstoyanov1@gmail.com>
Date: Sat, 3 Aug 2019 14:57:22 +0100
Subject: [PATCH] pylint: Specify string format as parameters

Pylint (logging-not-lazy) warning

Specify string format arguments as logging function parameters

Signed-off-by: Radostin Stoyanov <rstoyanov1@gmail.com>
---
 src/virtBootstrap/virt_bootstrap.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/virtBootstrap/virt_bootstrap.py b/src/virtBootstrap/virt_bootstrap.py
index f70acbd..b486ac3 100755
--- a/src/virtBootstrap/virt_bootstrap.py
+++ b/src/virtBootstrap/virt_bootstrap.py
@@ -151,7 +151,7 @@ def bootstrap(uri, dest,
     if root_password:
         if root_password.startswith('file:'):
             root_password_file = root_password[len('file:'):]
-            logger.debug("Reading root password from file %s" %
+            logger.debug("Reading root password from file %s",
                          root_password_file)
             with open(root_password_file) as pwdfile:
                 root_password = pwdfile.readline().rstrip("\n\r")
